package.json

```diff
@@ -43,5 +44,6 @@
       "TWITTER_ACCESS_TOKEN_KEY": "@twitter-access-token-key",
       "TWITTER_ACCESS_TOKEN_SECRET": "@twitter-access-token-secret"
     }
-  }
+  },
+  "devDependencies": {}
```

Comment:
oh right, sorry for this empty line, in fact I'd have done `yarn --dev add uglifyjs-webpack-plugin@1.0.0-beta.1` instead of `yarn add uglifyjs-webpack-plugin@1.0.0-beta.1`

Reply:
No problem. We will take care of that before the next release.